test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -0,0 +1,159 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import static seedu.addressbook.util.TestUtil.assertCommandResult;
+
+public class ViewCommandTest {
+    private AddressBook addressBook;
```

Comment:
Also, maybe name this `typicalAddressBook` to signal that it is filled with all the `TypicalPersons`.